### DISCUSSION TITLE
Compiled dlt viewer in my repository?

### DISCUSSION THREAD

**Discussion by trteodor** [COVESA/dlt-viewer#284] [General]
Hello,
I have short question to you because i'am not sure :( and I don't want to break any licenses

Can I add compiled sources (execute binaries) of DLT Viewer to my repository?
https://github.com/trteodor/ucDLTlib
Because I prepared a simple library to send dlt logs from MCU and want to add compiled DLT viewer for quick start

I ask because i don't see in your repository releases with executive binaries..

**Comment by alexmucde** [comment]
@trteodor Should be possible, please consider the terms of open source license of Qt.
